.typos.toml

```diff
@@ -0,0 +1,2 @@
+[default.extend-words]
+skeep = "skeep"
```

Comment:
I am not finding this word in a dictionary, what's going on here? ^^

Reply:
Hehe, it's `skip` but I don't to use the same name as `skip` to highlight it does a difference thing. If you don't like that, I can fallback to `skip`, not a big deal after all.